packages/account/src/Components/api-token/api-token-delete-button.tsx

```diff
@@ -30,15 +30,17 @@ const ApiTokenDeleteButton = ({ token, popover_alignment = 'left' }: TApiTokenDe
         if (!is_deleting && isDesktop()) setIsPopoverOpen(false);
     };
 
-    const handleNo = () => setIsDeleting(false);
+    const onCancel = () => setIsDeleting(false);
 
-    const handleYes = () => {
+    const onSubmit = () => {
         setIsLoading(true);
         deleteToken(token.token);
-        if (isMounted() && isSuccess) {
-            setIsLoading(false);
-            setIsDeleting(false);
-        }
+        deleteToken(token.token).finally(() => {
```

Comment:
Why are there 2 consecutive calls for `deleteToken()`? The function returns promise, I think u can remove **Line:37**. and change `onSubmit()` to async and await? Then u need not just have a `finally()`